requirements/pytorch/extra.txt

```diff
@@ -8,4 +8,4 @@ hydra-core >=1.0.5, <1.4.0
 jsonargparse[signatures] >=4.26.1, <4.27.0
 rich >=12.3.0, <13.6.0
 tensorboardX >=2.2, <2.7.0  # min version is set by torch.onnx missing attribute
-bitsandbytes <=0.41.1
+bitsandbytes ==0.41.1  # strict
```

Comment:
```suggestion
bitsandbytes >=0.41.1, <0.42.0  # strict
```

Reply:
0.41.2 has breaking changes so this suggestion would break it